Applications/DataExplorer/VtkVis/VtkConsoleOutputWindow.cpp

```diff
@@ -0,0 +1,87 @@
+/**
+*
+* \copyright
+* Copyright (c) 2015, OpenGeoSys Community (http://www.opengeosys.org)
+*            Distributed under a Modified BSD License.
+*              See accompanying file LICENSE.txt or
+*              http://www.opengeosys.org/project/license
+*
+*/
+#include "VtkConsoleOutputWindow.h"
+
+#include <string>
+
+#include "vtkObjectFactory.h"
+#ifdef WIN32
+#include "vtkWindows.h"
+#endif
+
+vtkStandardNewMacro(VtkConsoleOutputWindow);
+
+//----------------------------------------------------------------------------
+VtkConsoleOutputWindow::VtkConsoleOutputWindow()
+{
+
+}
+
+//----------------------------------------------------------------------------
+VtkConsoleOutputWindow::~VtkConsoleOutputWindow()
+{
+}
+
+//----------------------------------------------------------------------------
+// Display text in the window, and translate the \n to \r\n.
+//
+void VtkConsoleOutputWindow::DisplayText(const char* someText)
+{
+	if(!someText)
+		return;
+
+	// Disable warnings
+	std::string someTextString(someText);
+	if((someTextString.find(
+		"This is very expensive for vtkMappedDataArray subclasses, since the scalar array must be generated for each call.") != std::string::npos) ||
+		(someTextString.find("Invalid framebuffer operation") != std::string::npos))
+		return;
+
+	// Create a buffer big enough to hold the entire text
+	char* buffer = new char[strlen(someText)+1];
```

Comment:
I'd recommend to rewrite the algorithm using std::string, instead of char* and libc functions.